src/rust/engine/fs/src/glob_matching.rs

```diff
@@ -321,7 +327,7 @@ trait GlobMatchingImplementation<E: Display + Send + Sync + 'static>: VFS<E> {
           .unwrap_or_else(|| vec![])
       })
       .and_then(move |link_globs| {
-        future::result(PathGlobs::from_globs(link_globs))
+        future::result(PathGlobs::from_globs(link_globs, "".to_string()))
```

Comment:
As with the python side, if the behavior is `ignore`, the description should be safely None.

A nit: in rust, cases where two fields of one struct have dependencies on one another (particularly when one of the fields is an enum) can suggest moving one of the fields into an enum:
```
enum GlobMatchBehaviour {
  Warn(description_of_origin: String).
  Error(description_of_origin: String),
  Ignore,
}
```

ie: ignore holds no data.

----

Failing that, having the description be an `Option<String>` would make more sense than using the empty string here.

Reply:
Fantastic suggestion! I love this part of Rust. Thanks!